content/en/user-guide/aws/events/index.md

```diff
@@ -13,6 +10,10 @@ EventBridge provides a centralized mechanism to discover and communicate events
 
 LocalStack allows you to use the EventBridge APIs in your local environment to create rules that route events to a target. The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_events/), which provides information on the extent of EventBridge's integration with LocalStack. For information on EventBridge Pipes, please refer to the [EventBridge Pipes]({{< ref "user-guide/aws/pipes" >}}) section.
 
+{{< callout >}}
+We have introduced an all-new LocalStack-native EventBridge provider available behind a feature flag. You can activate it by configuring `PROVIDER_OVERRIDE_EVENTS=v2` in your LocalStack configuration.
+{{< /callout >}}
```

Comment:
we also need to set the java home folder with `JAVA_HOME=` because we exclusively use the new event ruler - we can also link here to the new event ruler`